### PR TITLE
LOAST Test Result Changes for Facilities

### PR DESCRIPTION
**Pull request by jnguy-va**
https://vajira.max.gov/browse/API-5933

### CODE REVIEW

File: facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/FacilitiesSearchApi.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ FacilitiesResponse getFacilitiesByLocation(
                   "Latitude of point to search for facilities, "
                       + "in WGS84 coordinate reference system.",
               schema = @Schema(type = "number", format = "float"),
-              examples = @ExampleObject(name = "coordinates", value = "40.0"))
+              examples = @ExampleObject(name = "coordinates", value = "123.4"))
```
Comment (taylorrassmannlibertyits) [inline facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/FacilitiesSearchApi.java:149]: This is an invalid latitude range

**Review by taylorrassmannlibertyits — DISMISSED** [review]

File: facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/GeoFacility.java [hunk 2]
```diff
@@ -124,7 +126,7 @@
     @JsonProperty(value = "detailed_services")
```
Comment (taylorrassmannlibertyits) [inline facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/GeoFacility.java:126]: Make sure to add nullable true here as well (already added in Facility.java)

**Comment by jnguy-va** [comment]
Looking at this...

File: facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/NearbyResponse.java [hunk 3]
```diff
@@ -17,7 +17,8 @@
 
   public enum Type {
     @JsonProperty("nearby_facility")
-    NearbyFacility
+    NearbyFacility,
+    nearby_facility
```
Comment (taylorrassmannlibertyits) [inline facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/NearbyResponse.java:21]: Why was this added? This means there is a property type we were missing before?

Comment (jnguy-va) [reply facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/NearbyResponse.java]: Removed this change with the latest push, but the LOAST testing tool is flagging that "nearby_facility" isn't a valid enum for Type.  Trying to make a work around, but so far I think it might be something on the test tool end that needs to support @JsonProperty within enums.

Comment (taylorrassmannlibertyits) [reply facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/NearbyResponse.java]: In that case I think you should remove this new enum value. Especially since it is unused.

**Review by taylorrassmannlibertyits — DISMISSED** [review]

**Comment by jnguy-va** [comment]
From the CMS Documentation PR: https://github.com/department-of-veterans-affairs/lighthouse-facilities/pull/208#issuecomment-817774002

> A quick note on the discussion of periods in schema descriptions (for consistency) - there is no internal consistency within the Facilities API with regard to periods appearing at the end of schema description lines. In fact, the vast majority do not have terminating periods prior to this update. May want to check for a company/program style guide prior to making sweeping changes.

I see the periods have been added to that PR, do we want them to also be cleaned up for the rest of Facilities here?

File: facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/NearbyResponse.java (hunk 3 above)
Comment (jnguy-va) [reply facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/NearbyResponse.java]: Sent some feedback to the LOAST team, waiting for reply if we need to fix or if it's on their end not supporting the JSON property name.

File: facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/Facility.java [hunk 4]
```diff
@@ -197,26 +202,31 @@
...
-    @Schema(description = "This field is deprecated and replaced with \"operating_status\".")
+    @Schema(
+        description = "This field is deprecated and replaced with \"operating_status\".",
+        nullable = true)
     ActiveStatus activeStatus;
 
     @Valid
     @NotNull
     @JsonProperty(value = "operating_status", required = true)
+    @Schema(example = "A")
```
Comment (as-va) [inline facilities-api/src/main/java/gov/va/api/lighthouse/facilities/api/v0/Facility.java:221]: This is not a valid example for `operating_status` - this is an example of an `active_status`. See the `OperatingStatus` class for a description the `operating_status` structure.

**Comment by jnguy-va** [comment]
There is currently a swagger bug with serializing enums with @JsonProperty or @JsonValue.

https://github.com/swagger-api/swagger-core/pull/3553
https://github.com/openapi-tools/swagger-maven-plugin/issues/52

**Review by as-va — APPROVED** [review]
